src/shared/utils.ts

```diff
@@ -1490,3 +1491,16 @@ export function newVote(voteType: VoteType, myVote?: number): number {
     return myVote == -1 ? 0 : -1;
   }
 }
+
+function sleep<T>(millis: number): Promise<T> {
```

Comment:
The generic isn't needed here. You could get away just fine returning `Promise<void>`.

Reply:
Sry, getting to this now.